src/main/resources/templates/db_scripts/schema.json

```diff
@@ -612,6 +634,13 @@
           "fieldName": "exportJobId",
           "caseSensitive": false
         }
+      ],
+      "ginIndex": [
+        {
+          "fieldName": "exportedPoLineIds",
+          "caseSensitive": false,
+          "removeAccents": true
+        }
```

Comment:
Resolves this WARN log:
`10:28:55 [924539/orders;230710/orders-storage] [diku] [c88a8880-f8ec-55c5-a808-14cfb0744108] [mod_orders_storage] WARN  CQL2PgJSON           Doing wildcard LIKE search without index for export_history.jsonb->>'exportedPoLineIds', CQL >>> SQL: exportedPoLineIds == "*\"293fc302-d1be-407b-a6f8-292fa7cb7d91\"*" >>> lower(f_unaccent(export_history.jsonb->>'exportedPoLineIds')) LIKE lower(f_unaccent('%"293fc302-d1be-407b-a6f8-292fa7cb7d91"%'))
`